docs/src/comparisons.md

```
@@ -1,52 +1,58 @@
+---
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: p

(MD033, no-inline-html)